File: src/main/java/mediathek/tool/SenderListComboBoxModel.java
package mediathek.tool;

import ca.odell.glazedlists.swing.DefaultEventComboBoxModel;

public class SenderListComboBoxModel extends DefaultEventComboBoxModel<String> {

    public SenderListComboBoxModel() {
        super(new EventListWithEmptyFirstEntry(SenderListBoxModel.getReadOnlySenderList()));
        setSelectedItem("");
    }
}

File: src/main/kotlin/mediathek/tool/SenderListComboBoxModel.kt
package mediathek.tool

import ca.odell.glazedlists.swing.DefaultEventComboBoxModel
import mediathek.config.Daten

class SenderListComboBoxModel(daten: Daten) : DefaultEventComboBoxModel<String>(
    EventListWithEmptyFirstEntry(daten.allSendersList)
) {
    init {
        selectedItem = ""
    }
}
